code/modules/holodeck/basketball.dm

```diff
@@ -0,0 +1,103 @@
+//Basketball items and structures
+
+/obj/item/toy/basketball
+	name = "basketball"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "basketball"
+	inhand_icon_state = "basketball"
+	desc = "Here's your chance, do your dance at the Space Jam."
+	w_class = WEIGHT_CLASS_BULKY //Stops people from hiding it in their bags/pockets
+
```

Comment:
sorry wesoda, i need a reason to give you a review, and you appear to have added an extra space, impeding the style of this file

perish